crc.py

```diff
@@ -504,7 +504,7 @@ def table(args):
     width = args.width
     polynomial = args.polynomial
     lookup_table = create_lookup_table(width, polynomial)
-    template = '0x{{:0<0{digits}X}}'.format(digits=width // 4)
+    template = '0x{{:0{}X}}'.format((width+3) // 4)
```

Comment:
why adding the `+3`? Currently only byte size widths are supported (may should be added to the docs)

Reply:
The +3 is when the bitlength of the polynomial is not a multiple of 4.

Reply:
Under isssue #26, it is mentioned that CRC9 works. I can imagine that CRC<8 will not work for 256-entry table-based implementation.